server/core/include/irods/notify_service_manager.hpp

```diff
@@ -0,0 +1,72 @@
+#ifndef IRODS_NOTIFY_SERVICE_MANAGER_HPP
+#define IRODS_NOTIFY_SERVICE_MANAGER_HPP
+
```

Comment:
This file needs the `/// \file` directive for doxygen to pick it up.

Reply:
Fixed